app/main.py

```diff
@@ -1 +1,21 @@
-# write your code here
+from app.cafe import Cafe
```

Comment:
If cafe.py and errors.py are in the same directory as main.py, importing should be done without the app. prefix unless the modules are nested within an app package.